### ISSUE TITLE
Accessibility Tests Failing

### ISSUE THREAD

**Issue by weiquu** [TEAMMATES/teammates#12647]
Not sure why the axe tests passed for #12645 but not for other PRs...

Anyway here seem to be the duplicate `id`s:
- `response-recipient`
- `response-giver`
- `comment-giver-name`
- `given-responses`

There seems to be an "`aria-label` attribute cannot be used on a `i` with no valid role attribute" error as well caused by the `rubric-answers` table

Will make a PR in a bit
